blockly-ws-search/workspace_search.js

```diff
@@ -146,7 +145,18 @@ class WorkspaceSearch {
     // Create the button
     var btn = document.createElement('button');
     Blockly.utils.dom.addClass(btn, className);
-    Blockly.bindEventWithChecks_(btn, 'click', this, onClickFn);
+    btn.addEventListener('click', onClickFn);
```

Comment:
I would rather leave that to the caller to decide since it allows more flexibility. For example if we want to create a btn but not bind it to this. Updated all the binds to use the arrow function instead of .bind.